aws/resource_aws_wafv2_ip_set.go

```diff
@@ -148,7 +151,7 @@ func resourceAwsWafv2IPSetRead(d *schema.ResourceData, meta interface{}) error {
 
 	resp, err := conn.GetIPSet(params)
 	if err != nil {
-		if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == wafv2.ErrCodeWAFNonexistentItemException {
+		if isAWSErr(err, wafv2.ErrCodeWAFNonexistentItemException, "AWS WAF couldn’t perform the operation because your resource doesn’t exist") {
```

Comment:
in case the returned error's `Message` string changes in the API, it'll be safer to omit the message string to match against
```suggestion
		if isAWSErr(err, wafv2.ErrCodeWAFNonexistentItemException, "") {
```